ownership-chain/runtime/src/migration.rs

```diff
@@ -0,0 +1,63 @@
+use frame_support::{
+	ensure,
+	traits::{Get, OnRuntimeUpgrade},
+	weights::Weight,
+};
+use pallet_evm::Pallet as Evm;
+use sp_core::H160;
+
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
+
+pub struct AddPrecompileBytecode<T>(sp_std::marker::PhantomData<T>);
+
+fn is_bytecode_stored<T: pallet_evm::Config>(address: H160) -> bool {
+	Evm::<T>::account_code_metadata(address).size != 0
+}
+
+impl<T> OnRuntimeUpgrade for AddPrecompileBytecode<T>
+where
+	T: pallet_evm::Config,
+{
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<Vec<u8>, sp_runtime::TryRuntimeError> {
+		let asset_metadata_extender_address: H160 = H160::from_low_u64_be(1029);
```

Comment:
could probably use `hash` function from the `precompile-utils`, to be more consistent